src/ragapp/patch/frontend/app/components/ui/lib/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { clsx, type ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs));
+}
+
+export function isValidUrl(url?: string): boolean {
```

Comment:
This is good to add in CL, should change to hasUrl() instead, and just use it directly here: https://github.com/run-llama/create-llama/blob/f04f60d555c9edf5f86d0d89a534b4c150887732/templates/types/streaming/nextjs/app/components/ui/chat/index.ts#L116

Reply:
yes, please try https://github.com/run-llama/create-llama/pull/259 - if it works, i'll release it - then you don't need to patch

Reply:
Please ship it! Thank you!